tools/legal-review/engine/org.eclipse.jgit.org.eclipse.jgit-6.3.0.202209071007-r/copyright-keep

```diff
@@ -0,0 +1,342 @@
+<li>Redistributions in binary form must reproduce the above copyright notice,
+<p>Copyright (c) 2007, Eclipse Foundation, Inc. and its licensors. </p>
+<p>Copyright (c) 2017:</p>
+<p>THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+<ul><li>Redistributions of source code must retain the above copyright notice,
+<ul><li>The above copyright notice and this permission notice shall be included
+ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR
+ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
```

Comment:
These are definitely wrong.

This adds lines like `ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR` to the copyright notice without any context. If we can do `keep with context` for them, that may make sense. If not possible, then please double-check if any new copyright info is present there (often this information has a lot of redundancy). If there is no new info, we can just ignore that note treating it as included in another one. If there is any new information, we should add it manually - the workflow to do so is to mark this entry as ignored, but then manually add additional information to a file `copyright-add` placed next to this one. Contents of such a file are appended to the NOTICE as-is.

Reply:
applied as discussed internally. All of those were already included in other cases, including the elaborate notice